src/instructlab/model/backends/backends.py

```diff
@@ -117,29 +158,29 @@ def determine_backend(model_path: pathlib.Path) -> Tuple[str, str]:
     """
     Determine the backend to use based on the model file properties.
     Args:
-        model_path (Path): The path to the model file.
+        model_path (Path): The path to the model file/directory.
     Returns:
         Tuple[str, str]: A tuple containing two strings:
                         - The backend to use.
                         - The reason why the backend was selected.
     """
-
-    if model_path.is_dir():
-        if sys.platform != "linux":
-            raise ValueError(
-                "Model is a directory containing huggingface safetensors files but the system is not Linux. "
-                "Using a directory with safetensors file will activate the vLLM serving backend, vLLM is only supported on Linux. "
-                "If you want to run the model on a different system (e.g. macOS), please use a GGUF file."
-            )
-        # If the model is a directory, it's a VLLM model - it's kinda weak, but it's a start
+    
+    if model_path.is_dir() and is_model_safetensors(model_path):
```

Comment:
Given that `is_model_safetensors` can raise something I think this needs to go in a try-expect block.

Reply:
added, thanks!